src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -113,7 +113,7 @@ public async Task UpdateSearchParameterStatusAsync(IReadOnlyCollection<string> s
             {
                 var searchParamUri = new Uri(uri);
 
-                var paramInfo = _searchParameterDefinitionManager.GetSearchParameter(searchParamUri);
+                SearchParameterInfo paramInfo = _searchParameterDefinitionManager.GetSearchParameter(searchParamUri);
```

Comment:
nit: was this change intended?

Reply:
Yes. I had initially edited that file with some changes that I later reverted. I left this one in since it seemed more readable.